definitions/synth-private_locations/definition.yml

```diff
@@ -0,0 +1,5 @@
+domain: SYNTH
+type: PRIVATE_LOCATION
```

Comment:
the folder is `synth-private_locations` it should probably be `synth-private_location`